apertium/mtx_reader.cc

```diff
@@ -39,11 +39,14 @@ MTXReader::MTXReader(VM &spec) :
 size_t MTXReader::pushSetConst(std::string &val)
 {
   size_t set_idx = spec.set_consts.size();
-  stringstream val_ss(val);
```

Comment:
Old code was fine and a bit more efficient.

Reply:
this was code that got moved from `std::string` to `UString` and then back